library/std/src/path.rs

```diff
@@ -2464,8 +2464,10 @@ impl Path {
     /// This function will traverse symbolic links to query information about the
     /// destination file. In case of broken symbolic links this will return `false`.
```

Comment:
There has also been some discussion of slightly rewording this sentence. But I'm less certain about this.